pkg/web_app/lib/src/screenshot_carousel.dart

```diff
@@ -37,8 +37,10 @@ void _setEventForScreenshot() {
   }
 
   void showImage(int index, UIEvent event) {
+    hideElement(imageElement!);
     event.stopPropagation();
```

Comment:
Why does it happen after the hideElement? Wouldn't it be better before it?

Reply:
The stop propagation? I think it might not matter that much, but I can swap the two lines.